packages/unmock-core/src/util.ts

```diff
@@ -12,3 +12,45 @@ export const doUsefulStuffWithRequestAndResponse = (
     `We will do amazing things with ${metaData} ${requestData} and ${responseData}.`,
   );
 };
+
+export const getAtLevel = (
+  nestedObj: any,
+  level: number,
+  filterCb?: (key: string, value: any) => boolean,
```

Comment:
To me this does not seem like a "callback" (as in call back when done) but more like a filtering function?